src/main/webapp/resources/js/components/Header/main-navigation/components/CartLink.jsx

```diff
@@ -38,14 +36,13 @@ export function CartLink() {
   }, []);
 
   return (
-    <Badge count={count}>
-      <Button
-        type="link"
-        className="t-cart-count"
-        href={setBaseUrl(`/cart/${inGalaxy ? "galaxy" : "pipelines"}`)}
-        icon={<IconShoppingCart data-count={count} />}
-        style={{ color: textColor }}
-      />
-    </Badge>
+    <a
+      style={{ padding: `0 ${SPACE_MD}` }}
```

Comment:
I'm assuming this is what is causing the cart link to display in a weird position on certain screen sizes when resizing?

Reply:
Fixed